lib/teselagen/api/tests/test_test_client.py

```diff
@@ -31,6 +31,19 @@
 @pytest.mark.incremental
 class TestTESTClient():
 
+    # TODO: figure out how to run a clean up fixture.
+    # @pytest.fixture(autouse=True, scope='module')
+    # def clean_up(
+    #     self,
+    #     logged_client: TeselaGenClient,
+    #     experiment: Dict[str, Any],
+    #     select_laboratory,
+    # ) -> None:
+
+    #     client = logged_client.test
+    #     experiment_id: str = experiment['id']
+    #     client.delete_experiment(experiment_id=experiment_id)
+
```

Comment:
Probably using a [yield fixture](https://docs.pytest.org/en/6.2.x/fixture.html#yield-fixtures-recommended)

I haven't checked this, but should probably be very similar to the following:
```suggestion
@pytest.fixture(autouse=True, scope="module")
def clean_up(
    self,
    logged_client: TeselaGenClient,
    experiment: Dict[str, Any],
    select_laboratory,
) -> None:
    # set-up
    # # here code any action needed before running the test
    
    # yield
    # # if this fixture yield a value, yield it here - otherwise, add an empty yield
    yield
    
    # tear-down
    # # here code the tear down, this will be run after a test is executed
    client = logged_client.test
    experiment_id: str = experiment['id']
    client.delete_experiment(experiment_id=experiment_id)
```

Reply:
Sure. Added tear down lines at the same *experiment* fixture